cypress/integration/pages/testsForAllPages.js

```diff
@@ -1,3 +1,4 @@
+import pathOr from 'ramda/src/pathOr';
```

Comment:
Please dont use ramda in the e2es, though it is pretty it raises the bar of comprehension for our testers.